rotkehlchen/utils/validation_schemas.py

```diff
@@ -0,0 +1,24 @@
+from marshmallow import Schema, fields
+
+from rotkehlchen.api.v1.encoding import UnderlyingTokenInfoSchema
+
+
+class AssetListSchema(Schema):
```

Comment:
We already have an `AssetSchema`. Use that.

If you still want to make its own class at least make it so that it has a post_load function and all of the processing you have in that big function is done there. At the end the post_load can just give you the input that goes to globaldb.add_asset.

```
                    asset_id=asset_data['identifier'],
                    asset_type=asset_type,
                    data=extra_information,
```